packages/quantic/force-app/main/default/lwc/quanticBreadcrumbManager/__tests__/quanticBreadcrumbManager.test.js

```diff
@@ -161,32 +220,195 @@ describe('c-quantic-breadcrumb-manager', () => {
         );
         const values = exampleFacetBreadcrumb.values.map((item) => item.value);
         expect(labels).toEqual(values);
+
+        const breadcrumbClearAllFiltersButton =
+          element.shadowRoot.querySelector(
+            selectors.breadcrumbClearAllFiltersButton
+          );
+        expect(breadcrumbClearAllFiltersButton).not.toBeNull();
+      });
+    });
+
+    describe('when the number of values exceeds the collapse threshold', () => {
```

Comment:
what do you mean here? for 1 specific facet or for many facets?

Reply:
The threshold is per 1 facet, if there's multiple values selected for the one facet